quickfixj-core/src/test/java/quickfix/MultiAcceptorTest.java

```diff
@@ -111,6 +129,26 @@ public void testMessageSentOnWrongAcceptor() throws Exception {
         }
     }
 
+    @Test
+    public void testContinueInitializationOnError() throws Exception {
+        testAcceptorApplication = new TestAcceptorApplication(3);
+        int freePort1 = AvailablePortFinder.getNextAvailable();
+        int freePort2 = AvailablePortFinder.getNextAvailable();
+        int freePort3 = AvailablePortFinder.getNextAvailable();
+
+        Acceptor acceptor = createAcceptor(freePort1, freePort2, freePort3, true);
+        acceptor.start();
+
+        List<SessionID> acceptorSessions = acceptor.getSessions();
+        assertEquals(2, acceptorSessions.size());
+        SessionID session1 = new SessionID(FixVersions.BEGINSTRING_FIX42, "ACCEPTOR-" + 1, "INITIATOR");
+        SessionID session3 = new SessionID(FixVersions.BEGINSTRING_FIX42, "ACCEPTOR-" + 3, "INITIATOR");
+        assertTrue(acceptorSessions.contains(session1));
```

Comment:
`assertFalse(acceptorSessions.contains(session2));` would be good.

Reply:
Thanks, done.